airflow/api_fastapi/core_api/routes/public/dag_sources.py

```diff
@@ -18,21 +18,20 @@
 
 from typing import Annotated
 
-from fastapi import Depends, Header, HTTPException, Request, Response, status
+from fastapi import Depends, HTTPException, Request, Response, status
+from itsdangerous import BadSignature, URLSafeSerializer
```

Comment:
```suggestion
from fastapi import Depends, HTTPException, Response, status
```

Reply:
I ran pre-commits locally and ruff gave this fix

Reply:
I think it cause by my manual resolve conflict when rebasing to latest main. Just resolve, thanks !